docs/manual/manual_redundant_throttle.rst

```diff
@@ -0,0 +1,113 @@
+##############################
+Redundant Throttle Support
+##############################
+
+***************************
+About Redundant Throttle
+***************************
+
+Vertiq's redundant throttle support is a safety feature that allows your module to accept multiple throttle sources' commands simultaneously, and to switch between them should 
+the primary source fail. For example, suppose you would like to control your module using the :ref:`DroneCAN protocol <dronecan_protocol>`. To do so, you connect your module's 
+CAN connections with your flight controller's, and connect no other throttle sources. 
+
+.. image:: ../_static/manual_images/redundant_throttle/basic_can_connection.png
+    :width: 50%
+
+Now, in flight, your CAN connection comes loose, and the module and flight controller lose DroneCAN communication. In this case, your vehicle will lose control as your module no longer 
+reacts to new throttle commands, and the module will eventually :ref:`manual_timeout`.
+
+Now, suppose that you connect DroneCAN as before, but now, also connect your module to a DSHOT output on your flight controller.
+
+.. image:: ../_static/manual_images/redundant_throttle/dshot_and_dronecan_connection.png
+    :width: 50%
+
+Under normal operating conditions, the module will apply only throttle commands received via DroneCAN (as determined by your redundant throttle configuration discussed :ref:`below <redundant_throttle_config>`). 
+If we once again sever the DroneCAN connection, however, the module will seamlessly transition to accepting DSHOT commands. By connecting multiple throttle sources, and 
+leveraging Vertiq's redundant throttle feature, you robustify your vehicle's ability to fly safely even after communication failures.

```

Comment:
On the one hand, it turns out robustify is actually a word from my googling, but on the other hand, it still feels weird to me. Maybe "you make your vehicle more robust against communication failures in flight" could work instead?

Reply:
ha...I did the same google search after I wrote it...I'll change it